tests/e2e/basic/client_connectivity_test/test_vlan_mode.py

```diff
@@ -340,15 +346,15 @@ def test_modify_ssid(self, request, update_ssid, get_lanforge_data, setup_profil
                                  debug_=False)
         staConnect.sta_mode = 0
         staConnect.upstream_resource = 1
-        staConnect.upstream_port = get_lanforge_data["lanforge_bridge_port"]
+        staConnect.upstream_port = get_lanforge_data["lanforge_vlan_port"]
         staConnect.radio = get_lanforge_data["lanforge_5g"]
         staConnect.resource = 1
         staConnect.dut_ssid = profile_data["ssid_name"]
         staConnect.dut_passwd = profile_data["security_key"]
         staConnect.dut_security = "wpa"
         staConnect.station_names = station_names
         staConnect.sta_prefix = get_lanforge_data["lanforge_5g_prefix"]
-        staConnect.runtime_secs = 30
+        staConnect.runtime_secs = 40
```

Comment:
What worries me here is that we have to change this into so many places. Can't this be a class method that is updating the staConnect attributes and some of them are default in the class?